pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "neofs-testlib"
-version = "1.1.15"
+version = "1.1.17"
```

Comment:
surely need double grade?

Reply:
Yes, there was a problem and I needed to manually update the version in pypi